selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/factory/TestFactory.java

```diff
@@ -26,28 +25,34 @@
   private final TestDashboardUrlProvider dashboardUrl;
   private final TestFactoryServiceClient testFactoryServiceClient;
   private final TestWorkspaceServiceClient workspaceServiceClient;
+  private final LoginPage loginPage;
 
   private final String factoryUrl;
 
   public TestFactory(
       String factoryUrl,
-      DefaultTestUser owner,
+      TestUser owner,
       FactoryDto factoryDto,
       TestDashboardUrlProvider dashboardUrl,
       TestFactoryServiceClient factoryServiceClient,
-      TestWorkspaceServiceClient workspaceServiceClient) {
+      TestWorkspaceServiceClient workspaceServiceClient,
+      LoginPage loginPage) {
```

Comment:
You moved TestFactory into the test module. Unfortunately we can't mix page objects and selenium core part. 
Describe usecase.